md_core/update_med_views/bot.py

```
@@ -0,0 +1,141 @@
+    text += '\n|}'
+    # ---
+    return text
+
+
+def make_views(languages, year, limit, maxv):
+    # ---
+    views = {}
+    # ---
+    for n, (lang, _) in enumerate(languages.items(), start=1):
+        # ---
+        if limit > 0 and n > limit:
```

Pylint (3.3.7)

[refactor] Simplify chained comparison between the operands

(R1716)